scripts/start-demo-with-sso.sh

```diff
@@ -0,0 +1,9 @@
+# Enable sso
+export ENABLE_SSO=true
+export OAUTH_ENABLED=true
+echo "$INFO Setting ENABLE_SSO=true..."
+echo "→ ENABLE_SSO=$ENABLE_SSO"
+echo "$INFO Setting OAUTH_ENABLED=true..."
+echo "→ OAUTH_ENABLED=$OAUTH_ENABLED"
```

Comment:
Should we merge those? Or deprecate one?

Reply:
Yes. `$OAUTH_ENABLED` controls camel routes authentication with OpenMRS. Setting `$OAUTH_ENABLED=$ENABLE_SSO` should work.